components/progresscircle/stories/progresscircle.stories.js

```diff
@@ -61,17 +62,60 @@ WithForcedColors.tags = ["!autodocs", "!dev"];
 WithForcedColors.parameters = {
 	chromatic: {
 		forcedColors: "active",
-		modes: disableDefaultModes
+		modes: disableDefaultModes,
 	},
 };
 
 export const StaticWhite = ProgressCircleGroup.bind({});
 StaticWhite.tags = ["!autodocs", "!dev"];
+StaticWhite.storyName = "Static white";
 StaticWhite.args = {
 	staticColor: "white",
 };
 StaticWhite.parameters = {
 	chromatic: {
-		modes: disableDefaultModes
+		modes: disableDefaultModes,
 	},
 };
+
+// ********* DOCS ONLY ********* //
+
+export const Sizing = (args, context) => Sizes({
+	Template: Template,
+	withHeading: false,
+	withBorder: false,
+	...args,
+}, context);
+Sizing.args = {};
+Sizing.tags = ["!dev"];
+Sizing.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * The indeterminate progress circle displays a repeating animation for the inner fill.
+ */
+export const Indeterminate = (args, context) => Sizes({
+	Template: Template,
+	withHeading: false,
+	withBorder: false,
+	...args,
+}, context);
+Indeterminate.args = {
+	isIndeterminate: true,
+};
+Indeterminate.tags = ["!dev"];
+Indeterminate.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const StaticWhiteDocs = Sizing.bind({});
+StaticWhiteDocs.tags = ["!dev"];
+StaticWhiteDocs.storyName = "Static white, indeterminate";
```

Comment:
Should we also have a non-indeterminate static white story for the docs page? The old docs site features both. What do you think?

Your current `StaticWhite` captures everything (like `Default`) from the test file, so I think that one would have to stay as is. But we could do a little refactoring to have a `StaticWhiteIndeterminateDocs` and `StaticeWhiteDocs`.

Reply:
I've added another story for it.